addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/Log4ShellScanRule.java

```diff
@@ -128,18 +128,22 @@ public void scan(HttpMessage msg, String param, String value) {
             ExtensionOast extOast =
                     Control.getSingleton().getExtensionLoader().getExtension(ExtensionOast.class);
             for (String attackPattern : ATTACK_PATTERNS) {
-                HttpMessage testMsg = getNewMsg();
-                Alert alert =
-                        newAlert()
-                                .setConfidence(Alert.CONFIDENCE_MEDIUM)
-                                .setParam(param)
-                                .setMessage(testMsg)
-                                .build();
-                String payload = extOast.registerAlertAndGetPayload(alert);
-                String attack = attackPattern.replace("{0}", payload);
-                alert.setAttack(attack);
-                setParameter(testMsg, param, attack);
-                sendAndReceive(testMsg);
+                try {
+                    HttpMessage testMsg = getNewMsg();
+                    Alert alert =
+                            newAlert()
+                                    .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                                    .setParam(param)
+                                    .setMessage(testMsg)
+                                    .build();
+                    String payload = extOast.registerAlertAndGetPayload(alert);
+                    String attack = attackPattern.replace("{0}", payload);
+                    alert.setAttack(attack);
+                    setParameter(testMsg, param, attack);
+                    sendAndReceive(testMsg);
+                } catch (Exception e) {
```

Comment:
Maybe `IOException`, which other exceptions are being thrown here?

Reply:
I think to be general with Exception make sense, because we want to continue with further payloads regardless of exception type, won't we?